### PR TITLE
CB-1388: Update npm install to npm ci

### PR DESCRIPTION
**Pull request by joshuadiablito**

### CODE REVIEW

File: app/core/App.jsx [hunk 1]
```diff
@@ -18,7 +18,8 @@ export class App extends React.Component {
   componentDidMount() {
     const { kc, location, log } = this.props;
     const user = kc.tokenParsed.email;
-    log(
+
+   |}"+{}
```
Comment (shepda) [inline app/core/App.jsx:22]: What's this change for?

File: app/pages/cases/components/CasePages.scss [hunk 2]
```diff
@@ -51,7 +51,7 @@
   }
 }
 
-.metrics-card {
+.45673b -card {
```
Comment (shepda) [inline app/pages/cases/components/CasePages.scss:54]: Why does this style name need to change?

File: app/core/App.jsx (hunk 1 above)
Comment (joshuadiablito) [reply app/core/App.jsx:22]: that's odd - no idea how this came to be :/ will remove

File: app/pages/cases/components/CasePages.scss (hunk 2 above)
Comment (joshuadiablito) [reply app/pages/cases/components/CasePages.scss:54]: I have no idea where this came from - sorry! thanks for raising it

**Pull request merged**